cmd/zpool/zpool_main.c

```diff
@@ -516,6 +530,319 @@ print_vdev_prop_cb(int prop, void *cb)
 	return (ZPROP_CONT);
 }
 
+/*
+ * Given a power string: "on", "off", "1", or "0", return 0 if it's an
+ * off value, 1 if it's an on value, and -1 if the value is unrecognized.
+ */
+static int zpool_power_parse_value(char *str)
```

Comment:
I think I'll keep it as-is, since it would correctly fail if the slot reported a power status of '2' or something.  Enclosures can report some weird values...